CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
```

Comment:
Use 
`[1.0.0-rc.0] - Unreleased`
as release name

Reply:
Is `1.x.x` the right tag, though? This might not be backwards compatible